### PR TITLE
Fix: Open location screen on locationSearch result

### PR DESCRIPTION
**Pull request by ua741**

### CODE REVIEW

File: lib/models/search/generic_search_result.dart [hunk 1]
```diff
@@ -1,12 +1,14 @@
+import "package:flutter/cupertino.dart";
```
Comment (ashilkn) [inline lib/models/search/generic_search_result.dart:1]: Why was this added?

**Review by ashilkn — APPROVED** [review]

File: lib/models/search/generic_search_result.dart (hunk 1 above)
Comment (ua741) [reply lib/models/search/generic_search_result.dart:1]: context
